installer/targets/ros-audio_capture/install.yaml

```diff
@@ -1,4 +1,5 @@
 - type: ros
   source:
-    type: system
-    name: audio-capture
+    type: git
+    url: https://github.com/ros-drivers/audio_common.git
```

Comment:
This needs to be reverted

Reply:
No I am working on the stream API. So I need it this way.

Reply:
Infact instead of `ros-drivers` it needs to be `tue-robotics`